trade-module/src/main/java/io/edpn/backend/trade/domain/service/FindCommodityService.java

```diff
@@ -0,0 +1,11 @@
+package io.edpn.backend.trade.domain.service;
+
+import io.edpn.backend.trade.application.dto.FindCommodityRequest;
+import io.edpn.backend.trade.application.dto.FindCommodityResponse;
+
+import java.util.List;
+
+public interface FindCommodityService {
+    
+    List<FindCommodityResponse> findCommoditiesNearby(FindCommodityRequest findCommodityRequest);
```

Comment:
personally, I would rename to a name in line with the usecase `findCommoditiesOrderByDistance`
```suggestion
    List<FindCommodityResponse> findCommoditiesOrderByDistance(FindCommodityRequest findCommodityRequest);
```